cairo-vm-cli/Cargo.toml

```diff
@@ -24,3 +24,4 @@ rstest = "0.17.0"
 default = ["with_mimalloc"]
 with_mimalloc = ["cairo-vm/with_mimalloc", "dep:mimalloc"]
 with_tracer = ["cairo-vm/with_tracer", "cairo-vm-tracer"]
+mod_builtin = ["cairo-vm/mod_builtin"]
```

Comment:
Why not just add the builtins to the base build?

Reply:
They are in the base build, what this feature does is adding them to the layouts. The builtin being added to the layout means that it will be created when running in proof_mode. This would generate differences when comparing outputs against the python VM which doesn't have these builtins yet, therefore we cannot add them to the base build